### PR TITLE
AARD-1671: Fission import panel modal

### PR DESCRIPTION
**Pull request by BrandonPacewic**
Added panel for local importing of mira files into the fission simulator.

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1671)

### CODE REVIEW

File: fission/src/aps/APSDataManagement.ts [hunk 1]
```diff
@@ -160,3 +160,27 @@ export async function getFolderData(project: Project, folder: Folder): Promise<D
         return undefined
     }
 }
+
+export async function downloadItem(project: Project, item: Item): Promise<void | undefined> {
```
Comment (BrandonPacewic) [inline fission/src/aps/APSDataManagement.ts]: ~This was leftover from the cleanup from Hunter's branch. Leaving it in here for now. Technically should be a part of the APS integration branch @PepperLola.~

Comment (BrandonPacewic) [reply fission/src/aps/APSDataManagement.ts]: Moved this section of code into `branp/hunters-download-item` for future use.

**Review by a-crowell — APPROVED** [review]
Works great!

**Comment by HunterBarclay** [comment]
@PepperLola I can't review this because I made it. Could you?

**Review by PepperLola — APPROVED** [review]
I was able to import the 2024 KitBot locally from my computer. Is this something to make tests for once Playwright is integrated?

**Pull request merged**